etc/portage/package.use/modules

```diff
@@ -0,0 +1,4 @@
+# Disable installation of man pages and shell setup files to be able to
+# install both Modules and Lmod.
+# Enable *conflict unload* and *require via* Modules features.
+sys-cluster/modules -shell-setup -man-install +new-features
```

Comment:
I think this is going to be an issue, since the compat layer installation script will create a `etc/portage/package.use` file with the contents from https://github.com/EESSI/compatibility-layer/blob/main/ansible/playbooks/roles/compatibility_layer/defaults/main.yml#L32. Maybe it would make sense to change that to a directory, so that fixed stuff like this can still be put in this repo, but it will require some work; it may be better to postpone that for a next EESSI version. As a quick solution it may be better to remove it from here and add it to the Ansible configuration.

Reply:
I have removed this file and made pull request EESSI/compatibility-layer#231 to add these installation options to the Ansible configuration.